src/pages/ProductDetail/components/Comment/Comment.tsx

```diff
@@ -0,0 +1,83 @@
+import React, { useState, useEffect, useCallback } from 'react';
+import styles from './Comment.module.css';
+import { useParams } from 'react-router-dom';
+import { postProductComment, getProductComments } from '../../../../api/itemApi';
+import CommentItem from '../CommentItem/CommentItem';
+
+interface Comment {
+    id: number;
+    writer: Writer;
+    content: string;
+    createdAt: string;
+    updatedAt: string;
+}
+
+interface CommentList {
+    nextCursor: number;
+    list: Comment[];
+}
+
+interface Writer {
+    image: string;
+    nickname: string;
+    id: number;
+}
+
+function Comment() {
+    const [value, setValue] = useState('');
```

Comment:
P3:
가능하면 어떤 타입인지 명시해주시는 것을 추천드립니다~
이렇게  처음에 생각한 타입을 명시하시면 가독성 측면에서도 좋고
typescript에서 추론되는 것보다 더 정확한 타입이라 실수를 줄이기도 좋습니다.

```suggestion
    const [value, setValue] = useState<string>('');
```